src/test/java/edu/cornell/kfs/paymentworks/service/impl/PaymentWorksUtilityServiceImplTest.java

```diff
@@ -72,6 +75,44 @@ public void convertFieldArrayToMapFromValues() {
 		assertEquals("the value was not what we expected", PAYMENT_WORKS_FIELD_FROM_VALUE, results.get(PAYMENT_WORKS_FIELD_NAME));
 	}
 	
+	@Test
+	public void shouldVendorBeSentToPaymentWorks1() {
```

Comment:
I think these unit test methods could be renamed to something a bit more meaningful like: activeDVVendorShouldBeSentToPaymentWorks